migrations/sql/V2020.03.05.12.47__mine_report_definition_unique_report_name_clean_up.sql

```diff
@@ -0,0 +1,2 @@
+--One time FIX from seed data bug inserting duplicate records. is fixed now.
+DELETE FROM mine_report_definition where mine_report_definition_id > 61;
```

Comment:
Would it be safer to delete "where not unique/distinct?" than "id > 61"? Unless you're certain that this case covers removing all the duplicated values.